schemas/com.snowplowanalytics.mobile/screen/jsonschema/1-0-0

```diff
@@ -0,0 +1,45 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a context that represents information pertaining to the current screen being viewed when an event occurs.",
+    "self": {
+        "vendor": "com.snowplowanalytics.mobile",
+        "name": "screen",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string",
+            "description": "The name of the screen viewed."
+        },
+        "type": {
+            "type": "string",
+            "description": "The type of screen that was viewed e.g feed / carousel."
+        },
+        "id": {
+            "type": "string",
+            "maxLength": 36,
+            "description": "An ID from the associated screenview event."
```

Comment:
Is it UUID?

Reply:
Yes. Do you think it's worth making it UUID instead?

Reply:
Yep, there's `format: uuid`

Reply:
just updated all fields where uuid applies